control_panel_api/serializers.py

```diff
@@ -58,6 +58,7 @@ class UserS3BucketSerializer(serializers.ModelSerializer):
     class Meta:
         model = UserS3Bucket
         fields = ('id', 'url', 'user', 's3bucket', 'access_level', 'is_admin')
+        read_only_fields = ('access_level',)
```

Comment:
Why do we have this `access_level` field if we don't even use in practice?
As I said in a separate comment being able to RW is independent from being an admin for an S3 bucket.

Reply:
I think we need to keep this for flexibility - we may want users who are not admins to be granted write access - possibly